packages/fuels-accounts/Cargo.toml

```diff
@@ -36,4 +37,4 @@ tokio = { workspace = true, features = ["test-util"]}
 [features]
 default = ["std"]
 coin-cache = ["tokio?/time"]
-std = ["fuels-core/std", "dep:tokio", "fuel-core-client/default", "dep:eth-keystore"]
+std = ["fuels-core/std", "dep:tokio", "fuel-core-client/default", "dep:fuel-core", "dep:eth-keystore"]
```

Comment:
It is worrysome to have `fuel-core` as a `std` dep.

`std` is enabled by default but `fuel-core-lib` (feature from `fuels`) is disabled by default.

Since `std` from `fuels` will enable `std` from `fuels-accounts` and that will enable `fuel-core`, we then compile `fuel-core` whether we use `fuel-core-lib` or not. That might not be ideal since the reason behind the feature was to cut down on compile times for demo-like environments.

Reply:
Thanks for the suggestion - I'll try forwarding the `fuel-core-lib` feature to `fuels-accounts` then